_includes/releases/release-downloads-docker-image.md

```diff
@@ -83,3 +83,8 @@ This release was withdrawn, and we've removed the links to the downloads and Doc
 $ docker pull {{ release.docker.docker_image }}:{{ release.version }}
 ~~~
 {% endif %}
+
```

Comment:
Tangential to this PR, but what if we move the testing release warning above the download links in the version-specific landing pages, like https://deploy-preview-16623--cockroachdb-docs.netlify.app/docs/releases/v23.1#v23-1-0-alpha-7? The admonition separates the links from the Docker image when logically they should all go together, and it makes it harder to find the Changelog heading for a given release, since multiple releases are on the same page.

Reply:
Yeah, that makes sense to me. I can bundle that into this PR if you want.

Reply:
I moved it anyway :)